src/fidesops/ops/api/v1/scope_registry.py

```diff
@@ -24,6 +24,8 @@
 )
 PRIVACY_REQUEST_REVIEW = "privacy-request:review"
 
+IDENTITY_VERIFICATION_READ = "identity-verification:read"
```

Comment:
Note to self, we should add this scope to fideslib.

Reply:
Do you have a ticket for this?